api_contracts/contracts/downloadCenter/DownloadCenter.md

```diff
@@ -3,7 +3,7 @@ HOST: https://api.usaspending.gov
 
 # Download Center
 
-These endpoints are used to power USAspending.gov's download center. 
+These endpoints are used to power USAspending.gov's download center. This data can be used to create account data files.
```

Comment:
`This data can be used to create account data files.` can be removed (might be leftover from a previous version of this file). We will eventually include custom award data download and data dictionary endpoints and will want to leave the top-level description more general.